tests/autoblocks/test_run_test_suite.py

```diff
@@ -32,14 +35,27 @@ def mock_cli_server_address_env_var():
         yield
 
 
-@dataclasses.dataclass()
+@dataclasses.dataclass
 class MyTestCase(BaseTestCase):
     input: str
 
     def hash(self) -> str:
         return self.input
 
 
+def expect_post_request(
```

Comment:
i will update the other tests to use this in another PR, didn't want the diff to be huge